server/asset/bip-id.go

```diff
@@ -3,594 +3,609 @@
+	715:      "sin",
+	768:      "ballz",
+	777:      "btw",
+	800:      "beet",
+	801:      "dst[dstra]",
+	808:      "qvt",
+	818:      "vet",
+	820:      "clo",
+	831:      "cruz",
+	852:      "desm",
+	886:      "adf",
+	888:      "neo",
+	889:      "tomo",
+	890:      "xsel",
+	900:      "lmo",
+	916:      "meta",
+	970:      "twins",
+	996:      "okp",
+	997:      "sum",
+	998:      "lbtc",
+	999:      "bcd",
+	1000:     "btn",
+	1001:     "tt",
+	1002:     "bkt",
+	1023:     "one[harmony]",
+	1024:     "ont",
+	1026:     "kex",
+	1027:     "mcm",
+	1111:     "bbc",
+	1120:     "rise",
+	1122:     "cmt[cybermiles]",
+	1128:     "etsc",
+	1145:     "cdy",
+	1337:     "dfc",
+	1397:     "hyc",
+	1524:     "taler",
+	1533:     "beam",
+	1616:     "elf",
+	1620:     "ath",
+	1688:     "bcx",
+	1729:     "xtz",
+	1776:     "l-btc",
+	1815:     "ada",
+	1856:     "tes",
+	1901:     "clc",
+	1919:     "vips",
+	1926:     "city",
+	1977:     "xmx",
+	1984:     "trtl",
+	1987:     "egem",
+	1989:     "hodl",
+	1990:     "phl",
+	1997:     "polis",
+	1998:     "xmcc",
+	1999:     "colx",
+	2000:     "gin",
+	2001:     "mnp",
+	2017:     "kin",
+	2018:     "eosc",
+	2019:     "gbt",
+	2020:     "pkc",
+	2048:     "mcash",
+	2049:     "true",
+	2112:     "iote",
+	2221:     "ask[permission]",
+	2301:     "qtum",
+	2302:     "etp",
+	2303:     "gxc",
+	2304:     "crp",
+	2305:     "ela",
+	2338:     "snow",
+	2570:     "aoa",
+	2718:     "nas",
+	2894:     "reosc",
+	2941:     "bnd",
+	3003:     "lux",
+	3030:     "xhb",
+	3077:     "cos",
+	3276:     "ccc",
+	3377:     "roi",
+	3381:     "dyn",
+	3383:     "seq",
+	3552:     "deo",
+	3564:     "dst[destream]",
+	4218:     "iota",
+	4242:     "axe",
+	5248:     "fic",
+	5353:     "hns",
+	5757:     "stacks",
+	5920:     "slu",
+	6060:     "go",
+	6666:     "bpa",
+	6688:     "safe",
+	6969:     "roger",
+	7777:     "btv",
+	8339:     "btq",
+	8888:     "sbtc",
+	8964:     "nuls",
+	8999:     "btp",
+	9797:     "nrg",
+	9888:     "btf",
+	9999:     "god",
+	10000:    "fo",
+	10291:    "btr",
+	11111:    "ess",
+	12345:    "ipos",
+	13107:    "bty",
+	13108:    "ycc",
+	15845:    "sdgo",
+	16754:    "ardr",
+	19165:    "safe[safecoin]",
+	19167:    "zel",
+	19169:    "rito",
+	20036:    "xnd",
+	22504:    "pwr",
+	25252:    "bell",
+	25718:    "chx",
+	31102:    "esn",
+	31337:    "thepowerio",
+	33416:    "teo",
+	33878:    "btcs",
+	34952:    "btt",
+	37992:    "fxtc",
+	39321:    "ama",
+	49344:    "stash",
+	65536:    "keth",
+	88888:    "ryo[c0ban]",
+	99999:    "wicc",
+	200625:   "aka",
+	200665:   "genom",
+	246529:   "ats",
+	424242:   "x42",
+	666666:   "vite",
+	1171337:  "ilt",
+	1313114:  "etho",
+	1313500:  "xero",
+	1712144:  "lax",
+	5249353:  "bco[ore]",
+	5249354:  "bhd",
+	5264462:  "ptn",
+	5718350:  "wan",
+	5741564:  "waves",
+	7562605:  "sem",
+	7567736:  "ion",
+	7825266:  "wgr",
+	7825267:  "obsr",
+	61717561: "aqua",
+	91927009: "kusd",
+	99999998: "fluid",
+	99999999: "qkc",
```

Comment:
These are the BIP-0044 changes from PR https://github.com/decred/dcrdex/pull/45.